docs/guides/operator-guide-azure-enclave.md

```diff
@@ -9,129 +9,312 @@ sidebar_position: 18
+   | `gatewaySubnetPrefix` | The vnet address prefix of the subnet that runs the UID2 Gateway.<br/>Default: `10.0.1.0/28` |
+
+2. Run the following command to trigger the deployment:
+
+   ```
+   az deployment group create --name vnet --resource-group {RESOURCE_GROUP_NAME} --parameters vnet.parameters.json  --template-file vnet.json
+   ```
+
+### Complete the UID2 Private Operator Setup
+
+The next step is to bring up multiple Azure Container Instances (ACIs) in the VPC subnetwork that you created.
+
+Follow these steps:
+
+1. Update the `operator.parameters.json` file with the following required values:
+
+   | Parameter | Description |
+   | :--- | :--- |
+   | `vaultName` | The name of the key vault for hosting the operator key secret. The value must match the name you created in [Complete Key Vault and Managed Identity Setup](#complete-key-vault-and-managed-identity-setup). |
+   | `deploymentEnvironment` | Indicates the environment you're deploying to: `integ` or `prod`. For details, see [Deployment Environments](#deployment-environments). |
+
+2. (Optional) If you don't want to accept the defaults, update the `operator.parameters.json` file with the following values. These parameters have default values and in most cases you won't need to make any updates.
+
+    Parameter | Description |
+   | :--- | :--- |
+   | `operatorKeyName` | The operator key secret name. The value must match the value specified in [Complete Key Vault and Managed Identity Setup](#complete-key-vault-and-managed-identity-setup). If you accepted the default, the value is `operator-key`. |
+   | `operatorIdentifier` | The name of the managed identity that will launch the container. The value must match the value specified in [Complete Key Vault and Managed Identity Setup](#complete-key-vault-and-managed-identity-setup). If you accepted the default, the value is `uid-operator`. |
+   | `vnetName` | The virtual network name. The value must match the value specified in [Set Up the VPC Network](#set-up-the-vpc-network). If you accepted the default, the value is `unified-id-network`. |
+   | `computeSubnetName` | The name of the subnet that will run the Private Operator. The value must match the value specified in [Set Up the VPC Network](#set-up-the-vpc-network). If you accepted the default, the value is `unified-id-subnet-operators`. |
+   | `count` | The count for the number of instances you want to bring up. The default is `2`. |
+
+2. Run the following command to deploy the ACIs:
+
+   ```
+   az deployment group create --name operator --resource-group {RESOURCE_GROUP_NAME} --parameters operator.parameters.json  --template-file operator.json
+   ```
+
+3. Get the IP addresses of the ACI instances you created by running the following command:
+
+   ```
+   az deployment group show -g {RESOURCE_GROUP_NAME} -n operator --query properties.outputs
+   ```
+
+   The output should look something like the following:
```

Comment:
Can we change this to something like:
The output will be a JSON string that includes the IP Addresses, in the following format:
{
      "ipAddress": {
        "type": "Array",
        "value": [
          "10.0.0.4"
        ]
      }
    }
The reason for this the output is large, and they will need to scroll to find this text. 

I suggest we only include 2 as the default in the parameters is 2 instances, so if they follow the defaults, there will only be 2